application/tests/unit/common/helpers/UtilsTest.php

```diff
@@ -129,7 +130,7 @@ public function testGetFrontendConfig()
 
         $params = \Yii::$app->params;
         $config = Utils::getFrontendConfig();
-        $this->assertEquals($params['idpName'], $config['idpName']);
+        $this->assertEquals($params['idpDisplayName'], $config['idpName']);
```

Comment:
As a parallel change to my my comment on Utils::getFrontendConfig(), should we change `$config['idpName']` here to `$config['idpDisplayName']`?